com.unity.render-pipelines.universal/Documentation~/2DRendererData_overview.md

```diff
@@ -24,7 +24,7 @@ This option is enabled by default. Clear this option to disable the Depth/[Stenc
 
 ## Camera Sorting Layer Texture
 
-The __2D Renderer Data__ specifies how Unity supplies the shader variable `CameraSortingLayerTexture` for use in custom shaders. You should use this data in the same frame, and on the following layers.
+The __2D Renderer Data__ specifies how Unity supplies the shader variable `CameraSortingLayerTexture` for use in custom shaders. You must use this data in the same frame, and on the following layers. Using CameraSortingLayerTexture before it has been capture may result in unexpected results.
```

Comment:
It is recommended that you use this data in the same frame and on the following layers, as using `CameraSortingLayerTexture` before it has been captured may result in unexpected results.